OpenRA.Platforms.Default/Shader.cs

```diff
@@ -135,7 +136,10 @@ public void Bind()
 			for (ushort i = 0; i < bindings.Attributes.Length; i++)
 			{
 				var attribute = bindings.Attributes[i];
-				OpenGL.glVertexAttribPointer(i, attribute.Components, OpenGL.GL_FLOAT, false, bindings.Stride, new IntPtr(attribute.Offset));
+				if (attribute.Type == ShaderVertexAttributeType.Float)
+					OpenGL.glVertexAttribPointer(i, attribute.Components, (int)attribute.Type, false, bindings.Stride, new IntPtr(attribute.Offset));
```

Comment:
```suggestion
					OpenGL.glVertexAttribPointer(i, attribute.Components, OpenGL.GL_FLOAT, false, bindings.Stride, new IntPtr(attribute.Offset));
```

this change can be undone. We know that it's a float